[prefers-color-scheme article] Add info about DevTools debugging and Puppeteer testing (#1904)
GoogleChrome/web.dev: src/site/content/en/blog/prefers-color-scheme/index.md

authors:
  - thomassteiner
date: 2019-06-27

> updated: 2019-11-18

Reviewer: ```suggestion
updated: 2019-11-25
```

  });
```

## Debugging and testing dark mode

### Emulating `prefers-color-scheme` in DevTools

Switching the entire operating system's color scheme can get annoying real quick,
so in Chrome DevTools, we now allow you to emulate the user's preferred color scheme

> in a way that only the currently visible tab is affected.

Reviewer 2: Slightly simpler:

```suggestion
so Chrome DevTools now allows you to emulate the user's preferred color scheme
in a way that only affects the currently visible tab.
```

Reviewer 2: maybe make "now allows you" a link to https://developers.google.com/web/updates/2019/10/devtools#userpreferences

> You can find the emulation setting in the Rendering drawer of the Chrome DevTools.

Reviewer: ```suggestion
Open the [Command Menu](https://developers.google.com/web/tools/chrome-devtools/command-menu), start typing `Rendering`, run the `Show Rendering` command, and then change the **Emulate CSS media feature prefers-color-scheme** option.
```

> You can find the emulation setting in the Rendering section of the Chrome DevTools.

Reviewer 2: ```suggestion
You can find the emulation setting in the Rendering drawer of the Chrome DevTools.
```

<figure class="w-figure">

>   <img src="devtools-emulate.png" alt="Emulate the CSS media feature prefers-color-scheme" width="945" height="652">

Reviewer: ```suggestion
  <img src="devtools-emulate.png" alt="A screenshot of the 'Emulate CSS media feature prefers-color-scheme' option that is located in the Rendering tab of Chrome DevTools" width="945" height="652">
```

>   <figcaption class="w-figcaption">Emulate the CSS media feature <code>prefers-color-scheme</code> in Chrome DevTools.</figcaption>

Reviewer: ```suggestion
```
Caption isn't needed here

  });
```

## Debugging and testing dark mode

### Emulating `prefers-color-scheme` in DevTools

Switching the entire operating system's color scheme can get annoying real quick,

> so in Chrome DevTools, we now allow you to emulate the user's preferred color scheme

Reviewer 2: This line should be removed now (sorry, GitHub wouldn't let me suggest a multi-line patch before)

Author: Ooops, fixed. Thanks!
